web/src/unread_ui.js

```diff
@@ -96,6 +97,7 @@ export function update_unread_counts(skip_animations = false) {
         $("#userlist-toggle-unreadcount"),
         res.right_sidebar_direct_message_count,
     );
+    inbox_ui.update();
```

Comment:
This should just be registered via `register_update_unread_counts_hook`, right?

Reply:
Yeah, done.